src/Build/Definition/ProjectLoadSettings.cs

```diff
@@ -48,6 +48,11 @@ public enum ProjectLoadSettings
         /// By default, evaluations performed via <see cref="Project"/> evaluate and collect elements whose conditions were false (e.g. <see cref="Project.ItemsIgnoringCondition"/>).
         /// This flag turns off this behaviour. <see cref="Project"/> members that collect such elements will throw when accessed.
         /// </summary>
-        DoNotEvaluateElementsWithFalseCondition = 32
+        DoNotEvaluateElementsWithFalseCondition = 32,
+
+        /// <summary>
+        /// Ignore faulted target files when evaluating the project
+        /// </summary>
+        IgnoreFaultedImports = 64,
```

Comment:
I like the name `IgnoreInvalidImports` better.

Reply:
Renamed to `Invalid`